blogposts/announcing-sourcegraph-3.6.md

```diff
@@ -0,0 +1,162 @@
+
+[**🔗 Add quick links to repositories or important resources below the search bar**](#)<br />
+
+[**☁️ Added support for Bitbucket Cloud external service configuration**](#)<br />
+
+[**🛠 Improved external service configuration and syncing experience for admins**](#)<br />
+
+[**🙅‍♀️ Easily exclude GitHub repositories from syncing (by regexp)**](#)<br />
+
+[**✈️ Meet us in person at GopherCon!**](#)<br />
+
+[**📝 Changelog**](#35-changelog)<br />
+
+[**🎖️ Thank you**](#thank-you)<br />
+
+</div>
+
+**Deploy or upgrade:** [Local](https://docs.sourcegraph.com/#quickstart-guide) | [AWS](https://github.com/sourcegraph/deploy-sourcegraph-aws) | [DigitalOcean](https://marketplace.digitalocean.com/apps/sourcegraph?action=deploy&refcode=48dfb3ccb51c) | [Kubernetes cluster](https://github.com/sourcegraph/deploy-sourcegraph)
+
+## Sourcegraph plugin provides native code intelligence to Bitbucket Server
+
+TO DO: VIDEO
+
+<!--
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/{ID}?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/{ID}" target="_blank">View on Vimeo</a></p>
+</p>
+-->
+
+Admins can now install the [Sourcegraph for Bitbucket Server plugin](https://github.com/sourcegraph/bitbucket-server-plugin) on their Bitbucket Server instance to provide code intelligence automatically to their teams. Code intelligence, previously available only through the browser extension, includes hovers, tooltips, and line decorations while browsing and reviewing code in Bitbucket Server.
+
+Previously, many users were not aware of the Sourcegraph browser extension. Now, admins can provide an easier onboarding path for all team members who want to browse or review code on Bitbucket Server.
+
+There are two configuration steps:
+
+1. Install the [Sourcegraph for Bitbucket Server plugin](https://github.com/sourcegraph/bitbucket-server-plugin)
+2. Update the [Bitbucket Server external service configuration](https://docs.sourcegraph.com/admin/external_service/bitbucket_server#native-extension) in your Sourcegraph instance to include `”corsOrigin”: “<bitbucket server url>”`
+
+## 30% indexed search latency improvement for small and medium instances
+
+![chart showing 30% improvement of search latency](images/3.6-search-latency.png "30% improvement of search latency")
+
+We have made several changes to our search algorithm that improve performance and reduce peak memory usage. Our benchmarks show that the **99th percentile latency** of indexed searches in small to medium instances (less than 3000 repositories) **is reduced by up to 30%**. We will be continuing work to further benchmark and optimize Sourcegraph for our customers with 30,000+ repositories.
```

Comment:
I can't think of a good alternative, but I don't like the "we made several changes" part because I instantly want to know **what** the changes were. This would be fine if we had a blog post that contained the changes and details but instead, can we link to an issue or PR which provides some level of detail?

Reply:
What about:

```suggestion
We have improved our search algorithm's performance and reduced peak memory usage. Our benchmarks show that the **99th percentile latency** of indexed searches in small to medium instances (less than 3000 repositories) **is reduced by up to 30%**. We will be continuing work to further benchmark and optimize Sourcegraph for our customers with 30,000+ repositories.
```

Reply:
Brilliant!